game-server/data/handlers/ai/instance/empyreanCrucible/PriestPreceptorElyosAI.java

```diff
@@ -0,0 +1,90 @@
+package ai.instance.empyreanCrucible;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import com.aionemu.gameserver.ai.AIName;
+import com.aionemu.gameserver.model.gameobjects.Creature;
+import com.aionemu.gameserver.model.gameobjects.Npc;
+import com.aionemu.gameserver.model.templates.npcskill.NpcSkillTargetAttribute;
+import com.aionemu.gameserver.skillengine.SkillEngine;
+import com.aionemu.gameserver.utils.ThreadPoolManager;
+import com.aionemu.gameserver.world.WorldPosition;
+
+import ai.AggressiveNoLootNpcAI;
+
+/**
+ * AI for Priest Preceptor (Elyos) in Empyrean Crucible
+ *
+ * @author w4terbomb
+ */
+@AIName("priest_preceptor_elyos")
+public class PriestPreceptorElyosAI extends AggressiveNoLootNpcAI {
+
+	private final AtomicBoolean startTask1 = new AtomicBoolean();
+	private final AtomicBoolean startTask2 = new AtomicBoolean();
```

Comment:
If you implement `HpPhases.PhaseHandler` and its `handleHpPhase` method like in the other AIs, you don't need this custom state management and don't have to override `handleAttack`.